rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/tsdb/20_mapping.yml

```diff
@@ -364,3 +364,52 @@ nested fields:
                 properties:
                   foo:
                     type: keyword
+
+---
+"Unable to define a metric type for a runtime field":
```

Comment:
Maybe it is worth having all tsdb/runtime fields tests in the same file `rest-api-spec/test/tsdb/120_runtime_fields.yml`

Reply:
This test was there but Nik suggested to move it here. So I am not sure now where this belongs.

Reply:
Are these runtime fields yaml tests also using tsdb?

Reply:
I'm ok moving the runtime fields TSDB tests to one spot if that makes everyone feel better.

Reply:
Honestly I had them together because there weren't many. But now that there are more pulling is fine.

Reply:
imo we also can re-organise the tsdb / runtime fields tests in a followup pr.

Reply:
Ok I will do that separately...moving all tests using runtime fields in a file `120_runtime_fields.yml`